[CRITICAL] Simple Docker Build fails ("requires Go 1.17")

### Description of the issue
Dockerfile based build fails

(Am I on the correct nkn repository? It feels strange that something so basic fails)

### Issue-Type (put a `x` sign in the square brackets)
- [x] bug report
- [ ] feature request
- [ ] Documentation improvement

### Your environment
* OS: win10 / docker desktop
* Go version
* Release tag/commit of the code: latest master

### Expected behaviour
This command (taken from README) should build

`docker build -f docker/Dockerfile -t nknorg/nkn .`

### Actual behaviour
```
PS P:\cry\nkn> docker build -f docker/Dockerfile -t nknorg/nkn .
[...]
#12 26.75 /go/pkg/mod/golang.org/x/sys@v0.4.0/unix/sysvshm_unix.go:33:7: undefined: unsafe.Slice
#12 26.75 note: module requires Go 1.17
#12 26.76 make[3]: *** [Makefile:29: build] Error 2
#12 26.76 make[3]: Leaving directory '/nkn'
#12 26.76 make[2]: *** [Makefile:71: build_local] Error 2
#12 26.76 make[2]: Leaving directory '/nkn'
#12 26.76 make[1]: *** [Makefile:75: build_local_with_proxy] Error 2
#12 26.76 make[1]: Leaving directory '/nkn'
#12 26.76 make: *** [Makefile:79: build_local_or_with_proxy] Error 2
------
executor failed running [/bin/sh -c make $build_args]: exit code: 2
PS P:\cry\nkn>
```

### Steps to reproduce

See above

(EDIT / UPDATE) 

### Workaround

changing the Dockerfile builder to `FROM golang:1.17-alpine as builder` seems to make it work:

https://github.com/nknorg/nkn/blob/master/docker/Dockerfile#L2

--- a/docker/Dockerfile
+++ b/docker/Dockerfile
@@ -1,5 +1,5 @@
 ARG base
-FROM golang:1.13.8-alpine as builder
+FROM golang:1.19-alpine as builder
 RUN apk add make git curl
 ADD . /nkn
 WORKDIR /nkn

